types/oidc-provider/index.d.ts

```diff
@@ -250,7 +250,7 @@ declare class Session extends BaseModel {
     static get(ctx: Koa.Context): Promise<Session>;
 }
 
-declare class Grant extends BaseModel {
+declare class Grant extends BaseToken {
```

Comment:
This is the only model that doesn't appear to match the old-provider library definitions for BaseToken vs BaseModel.